common/lib/xmodule/xmodule/assetstore/__init__.py

```diff
@@ -2,20 +2,26 @@
 Classes representing asset metadata.
 """
 
+from __future__ import absolute_import
+
+import json
 from datetime import datetime
+
 import dateutil.parser
 import pytz
-import json
+import six
 from contracts import contract, new_contract
-from opaque_keys.edx.keys import CourseKey, AssetKey
 from lxml import etree
-
+from opaque_keys.edx.keys import AssetKey, CourseKey
 
 new_contract('AssetKey', AssetKey)
 new_contract('CourseKey', CourseKey)
 new_contract('datetime', datetime)
-new_contract('basestring', basestring)
-new_contract('long', long)
+new_contract('basestring', six.string_types[0])
+if six.PY2:
+    new_contract('long', long)
+elif six.PY3:
```

Comment:
Could you just make the second case in these conditions an `else` clause?  I seriously hope we'll have ripped out usage of `six` before we switch to some future Python 4 release, but would rather not risk it.

Reply:
Done.